{% extends "base_main.html" %}

{% block head %}
{% load static %}

{% endblock %}


{% block content %}

<div class="row">
{% if request.user.is_authenticated %}
	<form method="post" enctype="multipart/form-data">
		{% csrf_token %}
		<p>Please upload a matrix file with columns as samples and rows as genes. The platform will predict the activities of signaling molecules in our collection.</p>
		
		<p>The input value separator should be the tab space, and the gene names should follow the standard gene symbols. The platform accepts text or Gzip files as input. Please see a <a href="/download/GSE147507.diff.gz/">sample file here</a>. The file size should be smaller than {{max_upload_size}}MB, and the number of samples should be less than {{max_sample_count}}. If you need to analyze more samples, please <a href="/download/" target="_blank">download</a> our software and run it locally.</p>
		
		<p>The expression values, from either RNASeq or MicroArray, should be transformed by log2(x+1). x could be FPKM, RPKM, or TPM for RNASeq. For single-cell RNASeq data, we used log2(TPM/10 + 1). We also recommend quantile-normalization across conditions. Some software package, such as RMA or DESeq, will automatically include all normalizations. We recommend input differential profiles between the two conditions. If data is from a sample collection without pairs, please mean-centralize the value of each gene across all samples.</p>
		
		{{ form.as_p }}
		<button type="submit">Submit</button>
	</form>

{% else %}
	<p>Please login first.</p>
{% endif %}
</div>

{% endblock %}
